src/components/login/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes, Component } from 'react'
+import { translate } from 'react-i18next'
+
+class Login extends Component {
+	constructor (props) {
+		super(props)
+		this._handleSubmit = this.handleSubmit.bind(this)
```

Comment:
Agreed on the `_` :) 
About the spec: It goes against the spec of the fat arrow functions. I wouldn't count too much on it being implemented :D